cvernier/cmssw
# update PAT tools and other files to be able to test deep double b model

Now the deep double-b model actually runs! Not sure if results are sensible though. Will check that next.

To test do this:
```bash
cd CMSSW_10_0_X_2017-12-18-1500/src
cmsenv
git pull # (after you merge this PR)
git clone https://github.com/jmduarte/RecoBTag-Combined -b deepdoubleb_v0 RecoBTag/Combined/data # (this copies the model from our branch which we can add to the main one with a PR once we see that it works)
scram b -j 12
cd RecoBTag/DeepFlavour/test
voms-proxy-init --voms cms # (to get your grid certificate)
cmsRun test_deep_doubleb_cfg.py
```
